driver-adapters/neon-lambda-basic/index.test.js

```diff
@@ -14,7 +14,15 @@ test('prisma version and output', async () => {
     Payload: '""',
   })
 
-  const { regResult, itxResult } = JSON.parse(JSON.parse(new TextDecoder().decode(response.Payload)).body)
+  let regResult, itxResult
+  try {
+    const parsed = JSON.parse(JSON.parse(new TextDecoder().decode(response.Payload)).body)
+    regResult = parsed.regResult
+    itxResult = parsed.itxResult
+  } catch (e) {
+    console.log('new TextDecoder().decode(response.Payload)', new TextDecoder().decode(response.Payload))
+    throw e
```

Comment:
nit: You could put the decoded result into a variable

Reply:
Also: does this change need to be ported to the other neon tests?

Reply:
>nit: You could put the decoded result into a variable
I can't assign it to the non-local variable then

>Also: does this change need to be ported to the other neon tests?
yes, I'm opening a separate PR after merging this

Reply:
see https://github.com/prisma/ecosystem-tests/pull/4092